presto-native-execution/presto_cpp/main/PrestoServer.cpp

```diff
@@ -852,6 +853,11 @@ std::vector<std::string> PrestoServer::registerConnectors(
       PRESTO_STARTUP_LOG(INFO) << "Registering catalog " << catalogName
                                << " using connector " << connectorName;
 
+      // TODO: Temporary Change to use HiveDataSource for Iceberg Connector
```

Comment:
Moving the discussion in #21584 to here:

Add an iceberg.properties in presto-native-execution/etc/catalog with the following
```
# The Presto "iceberg" catalog is handled by the hive connector in Presto native execution.
connector.name=hive

cache.enabled=true
```

Reply:
@yingsu00 I have added the catalog file in #21584 since the final changes are part of that PR